integration-test/src/test/java/com/datastrato/gravitino/integration/test/web/ui/utils/ChromeWebDriverProvider.java

```diff
@@ -106,7 +108,14 @@ private void downloadZipFile(String url, String zipFileName, String fileName) {
     try {
       LOG.info("Download the zip file from " + url + " to " + downLoadDir);
       File chromeDriverZip = new File(ChromeWebDriverProvider.downLoadDir, zipFileName);
-      FileUtils.copyURLToFile(new URL(url), chromeDriverZip);
+
+      URL fileUrl = new URL(url);
```

Comment:
<img width="1394" alt="image" src="https://github.com/datastrato/gravitino/assets/15794564/10eea942-0754-4495-b222-a6896dd32877">

`copyURLToFile` do have `conectionTimeout` and `readTimeout`, you don't need to change it.

Reply:
bingo!